src/index.js

```diff
@@ -122,7 +123,8 @@ class MiniCssExtractPlugin {
   constructor(options) {
     this.options = Object.assign(
       {
-        filename: '[name].css',
+        filename: DEFAULT_FILENAME,
+        filenameTemplate: () => options.filename || DEFAULT_FILENAME,
```

Comment:
Let's rename `filenameTemplate` to `moduleFilename` (example for wasm https://github.com/webpack/webpack/blob/a4e5f63d9d370b191bb9586570bc22bf2947e390/lib/wasm/WebAssemblyModulesPlugin.js#L65)

Reply:
good idea. done